docs/quick-start.md

```diff
@@ -57,7 +57,8 @@ This will serve the UI on <https://localhost:2746>. Due to the self-signed certi
 
 ## Install the Argo Workflows CLI
 
-Next, Download the latest Argo CLI from the same [releases page](https://github.com/argoproj/argo-workflows/releases/latest).
+Next, for easy use of Argo Workflows, it is recommended to install and use the Argo CLI.  
+You can check an overview of the installation and usage on the [Argo CLI Walk Through page](https://argoproj.github.io/argo-workflows/walk-through/argo-cli/).
```

Comment:
```suggestion
You can more easily interact with Argo Workflows with the [Argo CLI](https://argoproj.github.io/argo-workflows/walk-through/argo-cli/).
```
- [active voice](https://kubernetes.io/docs/contribute/style/style-guide/#use-active-voice) and [more direct](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)

Reply:
Thanks for noticing about #9412 and the style guide. I read your recommendations and agree. Removing the duplications and just adding simple install tip on Argo CLI page would be better.